spec/graphlient/webmock_client_query_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe 'App' do
+  include_context 'Dummy Client'
```

Comment:
I think this is unnecessary/potentially misleading.

I believe you only need this:

```ruby
  include Rack::Test::Methods
```

Check and also match in the docs?